quizer.py

```diff
@@ -38,6 +65,54 @@ def question_page():
     """
     Quiz question page - show question, handle answer.
     """
+    
+    if request.method == 'GET':
+        
+        
+        question = choice(session['user_questions'])
+        q_index = session['user_questions'].index(question)
+        
+        print question['question']
+        
+        session['start_time'] = datetime.datetime.now()
+        
+        return render_template('question.html', question=question, q_index=q_index)
+        
+    else:
```

Comment:
normalnie robi się odwrotnie a wiec `if request.method == 'POST':` a na `else` 'GET', bo w bloku dla POSTa używasz `request.form`. Jeżeli np ktoś wyśle jakiś egzotyczne żądanie (np. OPTIONS) to teraz dostanie wyjątek a gdyby warunek był odwrotnie to dostanie stronę identyczną z GET.